internal/cache/file/downloader/job_test.go

```diff
@@ -621,18 +621,8 @@ func (dt *downloaderTest) Test_Cancel_WhenDownlooading() {
 	AssertEq(nil, err)
```

Comment:
Do we need to wait for some time? We can directly cancel after the first job.Download call.